example/conf/webserv.toml

```
@@ -19,6 +19,11 @@ path = '/uploads'
 root = 'example/html'
 autoindex = 'on'
 
+[[server.location]]
+path = '/cgi'
+root = 'example/html'
+cgi_extensions = ['.cgi']
```

`/cgi/script.cgi` へのリクエストは `example/html/cgi/script.cgi` を実行するわけですが、`html/` 配下にあるべきじゃない。
あと、実装の置き場所がわかりやすいように `example/cgi/.gitkeep` 置いてほしい。

```suggestion
[[server.location]]
path = '/cgi'
root = 'example'
cgi_extensions = ['.cgi']
```